pyacq/core/manager.py

```diff
@@ -22,17 +23,40 @@ def create_manager(mode='rpc', auto_close_at_exit=True):
         process exits (only used when ``mode=='rpc'``).
     """
     assert mode in ('local', 'rpc'), "mode must be either 'local' or 'rpc'"
+    rpc_log.set_process_name('main_process')
+    
+    # make nice local log output
+    root_logger = logging.getLogger()
+    log_handler = rpc_log.RPCLogHandler()
+    # for some reason the root logger already has a handler..
+    while len(root_logger.handlers) > 0:
+        root_logger.removeHandler(root_logger.handlers[0])
+    root_logger.addHandler(log_handler)
+    
+    # Send local uncaught exceptions through logger for nice formatting
+    rpc_log.log_exceptions()
+        
+    # start a global log server
+    if rpc_log.get_logger_address() is None:
+        rpc_log.start_log_server(logger)
```

Comment:
It's really nice for debugging, especially because after it collects log records from all other processes, it sorts them by time before printing. Only issue is that different machines will have clock differences. I have a function to measure the clock diff, but haven't done anything with it yet..